explainer-server/public/javascripts/explain-editor-plain-js.js

```diff
@@ -43,13 +43,26 @@ function updateCheckboxState() {
  * Tag Search
  */
 
-function processCapiSearchResponse(response){
+
+function processCapiSearchResponseTags(divIdentifier,response,userInterfaceTagDescriptionKey){
+    response.results.forEach(function(tag){
```

Comment:
It might be worth having a go in this PR as the library looks relatively straightforwards, it has a useful `.withQueryParameter()` function as well.